### ISSUE TITLE
[BUG] The charge limit menu can't be changed

### ISSUE THREAD

**Issue by Sichroteph** [ayufan/steam-deck-tools#165]
The sub-menu for the charge limit (70% - 80% - 90% - 100%) seems to be fixed at 70% on my end. Any effort to adjust it to a different value doesn't seem to affect this setting.

By the way, this is a fantastic feature, thanks! I typically play in very short sessions (max 25 minutes), and this will greatly contribute to improving my battery life.

My setup : 1st gen Steam deck
Version : 7.2 (obviously)

**Comment by ayufan** [comment]
@Sichroteph Can you give me bios version?

**Comment by Sichroteph** [comment]
yes : F7A0105

**Comment by ayufan** [comment]
You need 107.

**Comment by Sichroteph** [comment]
thank you !

**Comment by Sichroteph** [comment]
I might have closed it too quickly. My Steam Deck BIOS version is now F7A0119, and the issue persists. Should I consider reinstalling the program?

**Comment by ayufan** [comment]
Nope. Will look later

**Comment by Sichroteph** [comment]
Did you have the chance to look at it maybe ? Do you think it is an issue on my side ?

**Comment by julolosvalles26** [comment]
My limit doesn't exceed 70 still charge up to 100
![1](https://github.com/user-attachments/assets/473db6cf-d28e-4d5f-93eb-8ed4c49ec73a)
![2](https://github.com/user-attachments/assets/9d50c479-b4a5-459c-ab30-d39da97c21c8)

**Comment by Sichroteph** [comment]
Yes, I still have this issue. My Steam Deck remains charged at 100% for months, and my sessions never last more than 30 minutes. This option would be very useful in extending the battery lifespan.
